pygmt/datasets/earth_relief.py

```diff
@@ -161,9 +161,13 @@ def load_earth_relief(
             )
     else:
         dataset_prefix = earth_relief_sources[data_source]
-
+    # Choose earth relief dataset
+    if data_source in ["igpp", "synbath"]:
+        dataset_name = "earth_igpp"
+    elif data_source in ["gebco", "gebcosi"]:
+        dataset_name = "earth_gebco"
```

Comment:
Rewrite use match-case statements:
```suggestion
    match data_source:
        case "igpp" | "synbath":
            dataset_name = "earth_igpp"
        case "gebco" | "gebcosi":
            dataset_name = "earth_gebco"
```